src/sys.c

```diff
@@ -742,6 +742,30 @@ DLLEXPORT jl_sym_t* jl_get_ARCH()
     return ARCH;
 }
 
+DLLEXPORT size_t jl_maxrss()
```

Comment:
is it worth using the libuv function directly from julia instead of this? (https://github.com/nodejs/node-v0.x-archive/blob/master/deps/uv/include/uv.h#L1020-L1039)